### ISSUE TITLE
Determination - verbatim

### ISSUE THREAD

**Issue by OwenLonsdale** [AAFC-BICoE/dina-planning#311]
The present determination fields are not adequate to capture all data on determination labels. A verbatim label data field is required. Examples:

1) “holotype ♂ Rhyacophila nyelungpa 1966 F. Schmid / HOLOTYPE CNC No. 11308 Rh. nyelunpa [sic].”
2) “Holotype CNC No. , Setodes 1-chringa, Vide altera pars.”
3) “HOLOTYPUS Coniopteryx canadensis n. sp. ♂ det. Meinander 1971/ HOLOTYPE CNC.No.12858 Coniopteryx canadensis Meinander”
4) “Chrysopa sp, near rufilabris may be new, Det. by RCSmith / Chrysopa placita Bks Det. P Adams / NEOTYPE Chrysopa intacta Navás GARLAND CNC No. 21287 / “Ceraeochrysa” intacta (Navás), in.sed. det. C.A. Tauber ’10”
5) “Order Odonata Suborder Anisoptera Family Corduliidae Neurocordulia michaeli Brunelle 2000 Holotype Male (specimen number 62) Determinor Brunelle P.M.”
6) “HOLOTYPE Sphaeropscoides canadensis Grimaldi & Engel, 2006 (Pscoptera [sic]: Sphaeropsocidae) Amer. Mus. Novit. 3523”
7) “N. jordani Hubbard April 1940 Det. 16.I.1942. Geo.P.Holland / vial 52 Nearctopsylla hygini R. ♂ vancouverensis ssp. n. Jul. Wagner det. / N.B. This is the Holo-type of N. hygini columbiana W. May ’40”
8) “Thrassis ♂ vancouverensis Stark 1957 HOLOTYPE PLAGUE LABORATORY COMMUNICABLE DISEASE CENTER, SAN FRANCISCO, CAL / HOLOTYPE 21247 CNC No.”

**Comment by dshorthouse** [comment]
Is this presently accommodated in the CNC db?

**Comment by OwenLonsdale** [comment]
Yes. We have a VERBATIM DETERMINATION field for all labels relating to dets. It must be carried over to DINA.

**Comment by dshorthouse** [comment]
Got it. I see a "identification" lexicon like this https://www.cnc.agr.gc.ca/taxonomy/Specimen.php?id=420586. The sequence of these (1., 2., 3., etc.) is unfortunately stored within the content of field; evidently no mechanism was designed to encode the ordering of these.
